docs/guides/email/02-basic-email-system.md

```diff
@@ -210,32 +232,30 @@ Mysql > insert into virtual_aliases(id,domain_id,source,destination) values(1,1,
 Mysql > insert into virtual_aliases(id,domain_id,source,destination) values(2,1,'all@mail.rockylinux.me','leeo@mail.rockylinux.me');
 ```
 
-Here I have not inserted the ciphertext password for the relevant email users, which requires the use of `doveadm pw -s SHA512-crypt -p twotestandtwo` command. See [here](#ap1)
+The password entries for the relevant email users are not shown here, as it requires the use of `doveadm pw -s SHA512-crypt -p twotestandtwo` command. See [here](#ap1)
 
 #### Knowledge of SHA512 (SHA-2)
 
-SHA-2 (Secure Hash Algorithm 2): A Cryptographic Hash function algorithm Standard. It is the successor to SHA-1.
-
-Main standards:
+The main password hashing algorithms are:
 
 * SHA-0
 * SHA-1
 * SHA-2: Contains these -- SHA-224, SHA-256, SHA-384, SHA-512, SHA-512/224, SHA-512/256
 * SHA-3
 
-In the SHA-2 encryption standard, the number in the algorithm refers to the digest length in bits.
+In the SHA-2 hashing standard, the number in the algorithm refers to the digest length in bits.
 
-It is well known that in RockyLinux 8 and other RHEL 8 variants, the algorithm used to encrypt user passwords is SHA-512.
+It is well known that in Rocky Linux 8 and other RHEL 8 variants, the algorithm used to encrypt user passwords is SHA-512.
 
 ```bash
-Shell(192.168.100.5) > grep -i method /etc/login.defs
+Shell(192.168.100.7) > grep -i method /etc/login.defs
```

Comment:
@jimcat8  :

-  what is the correct IP that should appear here -  192.168.100.5 (original) OR 192.168.100.7 (new) ?

Reply:
192.168.100.5